### ISSUE TITLE
Improve calculatePhysics performance

### ISSUE THREAD

**Issue by rom1504** [mhsjlw/flying-squid#100]
Currently spawning lot of entity (maybe 50), takes some amount of cpu (maybe 15%).
I think we can do better. An idea might be taking into account that the entity is on the ground to avoid recomputing collision too often.

**Comment by demipixel** [comment]
Also fix size > 1.0 blocks (I haven't tested it but I know it's not gonna be good).

**Comment by rom1504** [comment]
"An idea might be taking into account that the entity is on the ground to avoid recomputing collision too often." : I think we could just computing physics when onGround. And use an "blockChanged (x,y,z)" event to start back the computing of physics.

**Comment by demipixel** [comment]
in physics.js, on ground is "blockY" i.e. if (blockY) onGround == true

**Comment by rom1504** [comment]
Yes, but my point is if onGround is true, then you stop computing physics until there's a block that change under you. Oh and maybe adding the condition velocity == 0 .
